book/website/collaboration/inclusive-events/inclusive-events-comms.md

```diff
@@ -0,0 +1,487 @@
+
+(cl-inclusive-events-comms-speaker-preparation)=
+### Speaker Preparation
+Depending on the type of event you are planning, you may want to consider providing guidance for speakers, moderators, and presenters on inclusive communication, 
+and requesting that any content delivered in the session can be sent to attendees before the event. 
+
+If your event has online or streamed sessions, consider developing an information pack and working with your speakers and event facilitators to 
+ensure that content is accessible and inclusive of all attendees. 
+This information pack can contain guidance on how to present so that attendees with sensory processing differences or hearing impairments are able to engage in the session content. 
+For example, recommending that speakers remain as still as possible while they are speaking and presenting, and requesting that online presenters are well-lit with a blurred background 
+will help support attendees who rely on lip and face reading to engage in a session.
+
+You can also include information about the audience and provide guidance on how to effectively communicate to an audience of people with lived experience. 
+Many early career researchers may not have recieved formal training in scientific communication and working with the public, so it can be helpful to emphasize the importance of using plain language and speaking clearly. 
+
+You can also prepare the speakers by asking how they would like to interact with any questions that are asked. 
+Many research talks often follow a "lecture" format where a speaker talks on a topic and then takes 2-3 questions from the audience. 
+Consider asking your speaker to break their talk into sections to allow for questions at different points in the session, or if your event is a lecture, 
+consider using software to gather questions as the session progresses. 
+This can be helpful for people with lived experience of cognitive or sensory differences. 
+
+(cl-inclusive-events-comms-attendees)=
+## Who will be attending
+People with lived experience may experience additional anxiety due to being in an unfamiliar environment where they can often feel like their presence is tokenistic. 
+If you have planned your event well, attendees with lived experience should feel welcomed and able to engage fully in the event.
+
+It can be helpful in your communications to state who will be in attendance.
+For example "researchers, clinicians, policymakers, funding bodies, and people with lived experience". 
+
+Finally, consider including short introductions and background information about the speakers and session facilitators in your event information. 
+Many people with lived experience may feel more comfortable engaging in an event where they know who they are speaking with and it can be a helpful way to build rapport. 
+
+(cl-inclusive-events-comms-presentation-material)=
+## Presentation Slides and Event Content
```

Comment:
I think this has also two aspects: 
1. how the conference itself communicates in slides etc
2. supporting the presenters in making sure their slides are both psychologically & physically accessible

Maybe worth to split these aspects out?

Reply:
I made some changes in the most recent commit. Let me know what you think 🔥

Reply:
Yeah, that makes it a lot more concise! 👍